docs/index.rst

```diff
@@ -28,6 +28,9 @@ Baseline already Available
    DeepQSimple
    DuelQSimple
    SAC
+   DoubleDuelingDQN
+   DoubleDuelingRDQN
```

Comment:
it's an editorial choice, but can you move this model in the "more advanced baselines" ?